### ISSUE TITLE
[BITV] 9.4.1.2/7.6 - The "Log levels" button in the "Level" column header should be implemented as a button with an aria-expanded attribute.

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37134]
The current implementation using span elements is incorrect and inaccessible. Additionally, the individual menu items of the "Log levels" menu are within a-elements without an href attribute. The a-elements should be removed. (1)

- [ ] Fix
- [ ] Backport

![414c728086658dfdb595a873bd04deeb](https://user-images.githubusercontent.com/1315170/223749953-3818c168-c2cd-41ef-8cf1-83389591c617.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v7-n6

</details>
